public/content/documentation/web/component/tooltip.md

```diff
@@ -6,75 +6,112 @@ How to test a tooltip
 
-How to test a header
-
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Tab: Focus visibly moves to the tooltip button, and the tooltip appears.
+   - Spacebar: Activates the button (if applicable) or retains the tooltip visibility.
+   - Enter: Activates the button (if applicable) or retains the tooltip visibility.
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+   - Swipe: Focus moves to the tooltip button, tooltip content is announced.
+   - Doubletap: Activates the button (if applicable).
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
+   - Name: The accessible name of the tooltip button is clear.
+   - Role: It identifies as a button.
+   - Tooltip: The tooltip content is read aloud (via `aria-describedby` or role="tooltip").
+   - Action: It is clear whether the button performs an action or is static.
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/tooltip](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/tooltip)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
-
-GIVEN THAT I am on a page with a header landmark
+GIVEN THAT I am on a page with a tooltip button
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to a tooltip button  
+     - I SEE focus is strongly visually indicated  
+     AND I SEE a tooltip appears after a short delay
+   - THEN when I use the spacebar and/or enter key to activate the button  
+     I SEE the intended action occurs OR the tooltip remains visible without triggering an action
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
-
-   - I use the tab key to enter the web browser window
-
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - I HEAR It typically contains the name and primary navigation of the website
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver)  
+     AND I use the tab key to move focus to a tooltip button
+      - I HEAR the accessible name of the button
+      - I HEAR the tooltip content (if implemented using `aria-describedby` or role="tooltip")
+      - I HEAR whether the button is actionable or static
+   - THEN when I use the spacebar and/or enter key to activate the tooltip button  
+     I HEAR the intended action occurs (if applicable)
```

Comment:
This "I HEAR" statement needs a bullet and should be indented like the "I HEAR" statement above it.